src/nitni/nitni_base.nit

```diff
@@ -71,14 +71,15 @@ redef class MType
 
 	# Representation of this type in C for the internal of the system
 	# Hides extern types.
-	fun cname_blind: String is abstract
+	fun cname_blind: String do return "struct nitni_instance *"
 
 	# Representation of this type in mangled C
 	#   Object -> Object
 	#   Pointer -> Pointer
 	fun mangled_cname: String is abstract
 
-	# Does this types has a primitive reprensentation
+	# Does this types has a primitive representation?
```

Comment:
Just saw this is part of #1442, feel free to fix it or not since the PR is already ok will merge